spec/index.md

```diff
@@ -1322,6 +1329,7 @@ The following sections describe various System API functions, also referred to a
       ) -> ();
     ic0.call_on_cleanup : (fun : i32, env : i32) -> ();                         // U CQ Ry Rt CRy CRt T
     ic0.call_data_append : (src : i32, size : i32) -> ();                       // U CQ Ry Rt CRy CRt T
+    ic0.call_with_best_effort_response : (timeout_seconds : i32) -> ();         // U Ry Rt T
```

Comment:
Let's please also model those functions formally in the "Abstract Canisters to System API" section.